README.md

```diff
@@ -163,6 +163,15 @@ Build your own version:
 
 Check the result on `dist/` folder.
 
+Starting a local server:
+
+`npm run start`
```

Comment:
fyi, `npm start` is a shortcut for `npm run start`

https://docs.npmjs.com/cli/start